src/content-tags-drawer/ContentTagsCollapsible.test.jsx

```diff
@@ -141,19 +163,36 @@ describe('<ContentTagsCollapsible />', () => {
 
   it('should remove tag when they are unchecked in the dropdown', async () => {
     useTaxonomyTagsData.mockReturnValue({
-      isSuccess: true,
-      data: {
-        results: [{
+      hasMorePages: false,
+      tagPages: [{
+        isLoading: false,
+        isError: false,
+        data: [{
           value: 'Tag 1',
+          externalId: null,
+          childCount: 0,
+          depth: 0,
+          parentValue: null,
+          id: 12345,
           subTagsUrl: null,
         }, {
           value: 'Tag 2',
+          externalId: null,
+          childCount: 0,
+          depth: 0,
+          parentValue: null,
+          id: 12346,
           subTagsUrl: null,
         }, {
           value: 'Tag 3',
+          externalId: null,
+          childCount: 0,
+          depth: 0,
+          parentValue: null,
+          id: 12347,
           subTagsUrl: null,
         }],
-      },
+      }],
```

Comment:
Interesting library, I couldn't get to this yet unfortunately after trying to figure out the other issues I mentioned below. I will keep it in mind for future testing, unless you feel like it's a blocker for now then I can look into it? It would also require installing this devDependency since it's not already included in this repo.

Reply:
Nope, not a blocker.